neurovault/apps/statmaps/tests/test_celery.py

```diff
@@ -0,0 +1,31 @@
+# in one terminal window: 
```

Comment:
yes - probably this happened when you resolved conflicts

On Sat Jan 24 2015 at 3:15:06 PM vsoch <notifications@github.com> wrote:

> In neurovault/apps/statmaps/tests/test_celery.py
> <https://github.com/NeuroVault/NeuroVault/pull/111#discussion_r23500280>:
>
> > @@ -0,0 +1,31 @@
> > +# in one terminal window:
>
> I'm not sure about the image_details.html.haml, I don't think that I use
> it... should I delete?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NeuroVault/NeuroVault/pull/111/files#r23500280>.
>